boot/zephyr/CMakeLists.txt

```diff
@@ -129,6 +129,59 @@ if(DEFINED CONFIG_MEASURED_BOOT OR DEFINED CONFIG_BOOT_SHARE_DATA)
     )
 endif()
 
+# Include the sample implementation.
+if(CONFIG_MCUBOOT_UUID_VID OR CONFIG_MCUBOOT_UUID_CID)
+  zephyr_library_sources(
+    mcuboot_uuid.c
+    )
+endif()
```

Comment:
sample? If this is for a sample it should be added specifically by the sample, it should not be in the cmake file and it should not be added for all builds when these Kconfigs are enabled

Reply:
After second though - you're right: I cannot guarantee that this implementation will be sufficient for all use cases, thus it should not be placed inside MCUboot repository.